yaml/toy.yaml

```diff
@@ -0,0 +1,116 @@
+openapi: 3.0.3
+info:
+    title: Snapd REST API
+    license:
+        name: GPL-3.0
+        url: https://www.gnu.org/licenses/gpl-3.0.txt
+    description: |
+        The REST API provides access to snapd's state and many of its key functions,
+        as listed below.
+
+
+        For general information on how to use the API, including how to access it,
+        its requests and responses, results fields and error types, see [Using the
+        REST API](https://snapcraft.io/docs/using-the-api).
+    version: "0.1"
+servers:
+    - url: unix:///run/snapd.socket
+externalDocs:
+    url: https://snapcraft.io/docs
+    description: Snap and Snapcraft documentation
+paths:
+    /v2/aliases:
+        get:
+            operationId: getAliases
+            summary: Get the available app aliases
+            security: []
+            responses:
+                "200":
+                    description: A dictionary containing the aliases for each snap.
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    status-code:
+                                        type: integer
+                                        description: |
+                                            The status-code property contains the HTTP
+                                        enum:
```

Comment:
Since this is always the status code of a successful response, I think there's no other way.